src/applications/caregivers/components/PreSubmitInfo/index.jsx

```diff
@@ -116,11 +116,17 @@ const PreSubmitCheckboxGroup = ({ onSectionComplete, formData, showError }) => {
     - if hasSecondary one || two, first & last name must match, and be checked to submit
    */
 
+  console.log(
+    'showRepresentativeSignatureBox: ',
+    showRepresentativeSignatureBox,
+  );
```

Comment:
I don't think tests will pass with a `console.log` in the commit.

Reply:
🤦